rules/windows/discovery_system_service_discovery.toml

```
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2023/01/24"
+integration = ["windows", "endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/01/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects the usage of commonly used system service discovery techniques, which attackers may use during the reconnaissance phase 
+after compromising a system in order to gain a better understanding of the environment and/or escalate privileges. 
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "System Service Discovery through built-in Windows Utilities"
+risk_score = 21
+rule_id = "e0881d20-54ac-457f-8733-fe0bc5d44c55"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Discovery", "Elastic Endgame"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and
+  (
+  ((process.name: "net.exe" or process.pe.original_file_name == "net.exe" or (process.name : "net1.exe" and not process.parent.name : "net.exe")) and process.args : ("start", "use")) or
```

```suggestion
  ((process.name: "net.exe" or process.pe.original_file_name == "net.exe" or (process.name : "net1.exe" and not process.parent.name : "net.exe")) and process.args : ("start", "use") and process.args_count == 2) or
```

If we lock the args count to 2, we eliminate potential FPs from common uses of net.exe such as mounting drives, which typically has a arg count of 4. If args count is 2, then we are forcibly looking for `net use` or `net start` which are used to list all shared resources all services which I believe is more inline with the description of this rule. Thoughts?

---

Sounds fair to me. I was going to comment that that change would decrease visibility in case of "net start [service_name]" commands, but as this rule is only written to detect system service discovery, and not for stopping/starting services, I agree. Will commit your suggestion. Thanks!